### PR TITLE
Modify SQL queries to protect against SQL injections

### PR DESCRIPTION
**Pull request by sbtorres**
This PR modifies the way in which queries are executed to use parameters to protect against SQL injection attacks.

@weitsesun please review!

### CODE REVIEW

**Comment by sbtorres** [comment]
@weitsesun can you please review this one too? Thanks!

File: server/server.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const newRelic = require('newrelic');
```
Comment (weitsesun) [inline server/server.js]: Just FYI, if you are not using variable newRelic in server.js. Your can change your 1st line to `require('newrelic')`

File: k6POST.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    {duration: "270s", target: 150},
+  ]
+};
+
+// skew requests towards end of database per Learn Requirement
+let min = 8000000;
+let max = 10000000;
+const fakeText = ['dksalfjdasl ewqr d aic aewqr aka da', 'afkldfq caskl qeri das fc al', 'jdakl feqlk fda ca ewq', 'fjdkal eq pc qipurexnm'];
+const fakeDates = ['2019-01-01', '2018-10-10', '2019-04-04', '2019-05-01'];
+const fakeBooleans = ['t', 'f'];
```
Comment (weitsesun) [inline k6POST.js]: I have the same problem using faker in k6 script. I can only manually set fake data but using faker in this file. Let me know if you have figured out how to do it.

Comment (sbtorres) [reply k6POST.js]: For the fake data I don't think it really matters in this case right? The goal of the K6 stress testing is to test the latency of the POST request, not add real information to the database. That's why I just went with a few options. I delete them from my database each day after I run tests with a simple DELETE sql query.

File: server/server.js (hunk 1 above)
Comment (sbtorres) [reply server/server.js]: Oh nice, didn't realize that! I am going to remove it when testing my proxy server, so I will leave it for now. Good catch though!

**Pull request merged**